blazor/spreadsheet/accessibility.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Accessibility in Blazor Spreadsheet | Syncfusion
+description: Checkout and learn here all about accessibility in Syncfusion Blazor Spreadsheet and much more.
```

Comment:
Checkout and learn here all about accessibility in Syncfusion Blazor Spreadsheet component and much more